client/jetpack-connect/style.scss

```diff
@@ -656,3 +656,25 @@
 .jetpack-connect__navigation {
 	text-align: center;
 }
+
+.jetpack-connect__plan-info {
+	margin-top: 35px;
+	text-align: center;
+	width: 970px;
+}
+
+.jetpack-connect__plan-info-illustration {
+	margin: 30px 0 10px;
+	max-width: 25%;
+}
+
+.jetpack-connect__plan-info-buttons {
+	display: block;
+	margin-bottom: 20px;
+	text-align: center;
```

Comment:
This line is not necessary, this `div` will inherit the `text-align` from its parent.